tests/components/ozw/test_config_flow.py

```diff
@@ -18,9 +18,9 @@ async def test_user_create_entry(hass):
     assert result["errors"] is None
 
     with patch(
-        "homeassistant.components.zwave_mqtt.async_setup", return_value=True
+        "homeassistant.components.ozw.async_setup", return_value=True
     ) as mock_setup, patch(
-        "homeassistant.components.zwave_mqtt.async_setup_entry", return_value=True,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5518609)